cardano-api/src/Cardano/Api/TextView.hs

```diff
@@ -73,6 +74,13 @@ data TextView = TextView
   , tvRawCBOR :: !ByteString
   } deriving (Eq, Show)
 
+instance ToJSON TextView where
+  toJSON (TextView (TextViewType tvType) (TextViewTitle title) rawCBOR) =
```

Comment:
One thing that @dcoutts has suggested in the past is that we replace the `tvType` and `tvTitle` fields with a field such as `tvAttributes :: Map Text Aeson.Value` or `tvAttributes :: Aeson.Object`. Although, I think we can pass back over this in another PR and make those changes.

Reply:
May as well do it in this PR otherwise I have to regenerate all the golden files again